access/api.go

```diff
@@ -57,6 +58,7 @@ func TransactionResultToMessage(result *TransactionResult) *access.TransactionRe
 		StatusCode:   uint32(result.StatusCode),
 		ErrorMessage: result.ErrorMessage,
 		Events:       convert.EventsToMessages(result.Events),
+		BlockId:      result.BlockID[:],
```

Comment:
I guess we return a block ID equal to `0x0..0` when the block is not found, right?